cumulus/client/cirrus-executor/src/processor.rs

```diff
@@ -185,15 +192,38 @@ where
 		};
 		(&*self.client).import_block(block_import_params, Default::default()).await?;
 
+		let mut roots = self
+			.runtime_api
+			.runtime_api()
```

Comment:
`self.runtime_api.runtime_api()` looks wrong, maybe `self.runtime_api` should have been `self.client` instead?

Reply:
Nope. `self.runtime_api` was in cumulus bounded to the runtime API traits only, I later added `self.client` for our own usage, but it does look like `self.runtime_api` can be merged into `self.client` to reduce the generic parameter. I can add a TODO and refactor this later.

Reply:
TODO added.